app/views/checkout/_summary.html.haml

```diff
@@ -1,35 +1,29 @@
 %orderdetails
-  = form_for current_order, url: "#", html: {"ng-submit" => "purchase($event, checkout)"} do |f|
```

Comment:
@mkllnk but does this actually still need to be a form? The main submit control was commented out back in [this commit](https://github.com/openfoodfoundation/openfoodnetwork/commit/d227fb24fe43a06d36343a56f75c6d06b897249e#diff-ef94bd3a5e3be73decff09fb413f1297). (unless I'm not understanding quite how this is working...)